packages/nl-design-system-unstable/__docs__/inclusive-design.md

```diff
@@ -0,0 +1,13 @@
+# Inclusive design
+
+By placing diversity at the centre of your design process, your solution will ultimately serve a larger group of people.
+
+Exclusion happens when we solve problems using our own biases. Inclusive Design is a way to provide solutions that are useful and enjoyable for a diversity of people. This means including and learning from people with a range of perspectives.
+
+## Cultural differences
+
+### Color perception
+
+### Icon interpretation
+
+## Gender inclusion
```

Comment:
I feel the perceived quality of the documentation would benefit from not including clearly unfinished pages that are in this very early stage of development. And one essential thing is missing: the page is not yet included in the Storybook navigation. I will move this one page to a new branch so we can expand upon it before merging.

Reply:
The file is available again in the branch `docs/inclusive-design`.